config/taxonomy/application.values.schema.json

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "application.values.schema.json",
+    "title": "Application Values Taxonomy",
+    "description": "Values contributed to by the mesh for data applications.",
+    "definitions": {
+        "intent": {"$ref": "policymanager.structs.schema.json#/definitions/intent"},
+        "role": {"$ref": "policymanager.structs.schema.json#/definitions/intent"},
+        "app_info": {
+            "type": "object",
+            "properties": {
+                "intent": { "$ref": "#/definitions/intent" },
```

Comment:
this is double reference, why not directly reference policymanager.structs.schema.json# ?

Reply:
Fixed